src/rgw/rgw_common.h

```diff
@@ -1444,7 +1459,7 @@ struct RGWBucketInfo {
   RGWObjectLock obj_lock;
 
   void encode(bufferlist& bl) const {
-     ENCODE_START(20, 4, bl);
+     ENCODE_START(21, 4, bl);
```

Comment:
there's a corresponding `DECODE_START_LEGACY_COMPAT_LEN_32(20, 4, 4, bl);` on decode() that should be updated too

Reply:
oops, that would do it